code/game/machinery/doors/airlock.dm

```diff
@@ -1016,6 +1044,24 @@ About the new airlock wires panel:
 	if(!forced)
 		if( !hasPower() || isWireCut(AIRLOCK_WIRE_OPEN_DOOR) )
 			return 0
+	if(sabotaged && sabotageTurf)
+		p_open = 1
+		update_icon()
+		visible_message("<span class='warning'>[src]'s panel is blown off in a spray of deadly shrapnel!</span>")
+		explosion(sabotageTurf,-1,1,1)

```

Comment:
I'll see what I can do about your concerns, phil. however, there are two issues for me:

1. Disarming it easily. It's not worth anything if it can be disarmed without a hitch and I think there should at least be a fail chance that causes it to go off.
2. It not exploding on the sabotage turf. I'll most likely just make the door explode.